### PR TITLE
Feat/homescreen settings

### PR DESCRIPTION
**Pull request by builderjer**
This adds a sample `settings.json` to the homescreen page.  Also [this PR](https://github.com/OpenVoiceOS/skill-ovos-homescreen/pull/62) is mentioned and can be removed from the docs when applied

### CODE REVIEW

File: docs/homescreen.md [hunk 1]
```diff
@@ -49,4 +49,33 @@ The alarm widget is displayed in top left corner after the timer widget. It will
...
+
+## Configuration
+
+### Settings
+
+The homescreen has several customizations available.  This is sample `settings.conf` file with all of the options explained
+
+```
+{
+    "__mycroft_skill_firstrun": false,  # This is set on first load of the skill
```
Comment (NeonDaniel) [inline docs/homescreen.md:62]: With settings being `json`, I'm not sure if the comments are valid here?

File: docs/homescreen.md [hunk 2]
```diff
@@ -49,4 +49,33 @@ The alarm widget is displayed in top left corner after the timer widget. It will
...
 
-![](https://github.com/OpenVoiceOS/ovos_assets/raw/master/Images/homescreen-mediawidget.gif)
\ No newline at end of file
+![](https://github.com/OpenVoiceOS/ovos_assets/raw/master/Images/homescreen-mediawidget.gif)
+
+## Configuration
+
+### Settings
+
+The homescreen has several customizations available.  This is sample `settings.conf` file with all of the options explained
```
[the remark this answers is not in the document]
Comment (builderjer) [reply docs/homescreen.md:58]: You are right here, will change that soon

File: docs/homescreen.md (hunk 1 above)
Comment (builderjer) [reply docs/homescreen.md:62]: Comments are there because it is depreciated. I will move them out of the code

**Review by NeonDaniel — APPROVED** [review]

File: docs/homescreen.md (hunk 1 above)
Comment (NeonDaniel) [reply docs/homescreen.md]: current format look good to me

File: docs/homescreen.md [hunk 3]
```diff
@@ -49,4 +49,34 @@ The alarm widget is displayed in top left corner after the timer widget. It will
...
+    "wallpaper": "default.jpg",
+    "persistent_menu_hint": false,
+    "examples_enabled": true,
+    "randomize_examples": true,
+    "examples_prefix": true
+}
+```
+
+  * __mycroft_skill_firstrun:  This is automatically set on first load of skill
+  * weather_skill:  DEPRECIATED and has no effect - [PR pending](https://github.com/OpenVoiceOS/skill-ovos-homescreen/pull/62)
```
Comment (JarbasAl) [inline docs/homescreen.md:75]: typo: DEPRECATED (no longer supported) vs DEPRECIATED (down in value)

Comment (builderjer) [reply docs/homescreen.md]: Fixed this

**Pull request merged**